eclide/RepositoryFrame.h

```diff
@@ -86,7 +89,8 @@ class CRepositoryFrameT :	public CRepositorySlotImpl,
         m.EnableMenuItem(ID_REPOSITORY_OPEN, state.CanOpen ? MF_ENABLED : MF_GRAYED);
         m.EnableMenuItem(ID_REPOSITORY_OPENBUILDER, state.CanOpenBuilder ? MF_ENABLED : MF_GRAYED);
         m.EnableMenuItem(ID_EDIT_LOCATEFILEINEXPLORER, state.CanOpenInExplorer ? MF_ENABLED : MF_GRAYED);
-        m.EnableMenuItem(ID_REPOSITORY_COPY, state.CanCopy ? MF_ENABLED : MF_GRAYED);
+        m.EnableMenuItem(ID_REPOSITORY_COPY, MF_ENABLED);
```

Comment:
Isn't `state.CanCopy` still needed?

Reply:
@GordonSmith you are correct. Correcting.